extern/gallery/en-US/GalleryContents.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
Hi @Sancerio, can you make sure you update `Dynamo\tools\install\ExtractResources.bat` to copy this into `wwl` folder? Thanks!